packages/mx-token-exchange/README.md

```diff
@@ -10,7 +10,3 @@ This example project, built using [Next.js](https://nextjs.org), demonstrates ho
 2. Rename `.env.local.example` to `.env.local`, and enter the necessary access keys for both Dwolla and MX.
 3. Run `pnpm install` to download all necessary dependencies.
 4. Run `pnpm dev` to start the Next.js application! (Before connecting a bank account, please see _[Using MX Connect](#using-mx-connect)_.)
-
-## Using MX Connect
```

Comment:
Do we want to include anything about [test credentials](https://docs.mx.com/resources/test-platform)? I think we might do this on the other integration example sample apps READMEs

Reply:
Oh that's a good call! I just assumed any/all creds would work. I also forgot to remove the `(#using-mx-connect)` tag 🤦‍♀️ 

Addressed in the latest commit - [9445795](https://github.com/Dwolla/integration-examples/pull/30/commits/94457954255b8987c97aca14bed86795d65135e5)! 👍